content/authoring/guidelines/submission-tests.md

```diff
@@ -79,11 +81,11 @@ Random tests are uncommon in "real life" coding and are somewhat specific to Cod
 Some kata require solutions to be fast enough. For example, the author may only wish to accept solutions completing in (sub-)linear time. Building such test suites is not an easy task!
 
 - **Performance tests can be implemented in terms of random tests, by testing with large random inputs.** However, to make debugging easier, it can be worth having a separate set of random tests with small inputs first.
-- If the difficulty of a kata is roughly proportional to the size of the input, it's usually **better to have a few tests with large input rather than many with medium-sized inputs.** For example, 100 tests with huge numbers or arrays is usually better than 1000 tests with moderately large arrays/numbers (but see remarks on size of inputs above and the problem below about building the inputs too).
+- If possible, **performance of a solution should be evaluated by gauging the size of the inputs, rather than amount of tests.** If the difficulty of a kata is roughly proportional to the size of the input, it's usually better to have a few tests with large input rather than many with medium-sized inputs. For example, 100 tests with huge numbers or arrays is usually better than 1000 tests with moderately large arrays/numbers. At the same time, authors should ensure that large inputs are generated and handled in a way which does not have too much negative impact on the test suite.
```

Comment:
```suggestion
- If possible, **performance of a solution should be evaluated by gauging the size of the inputs, rather than the amount of tests.** If the difficulty of a kata is roughly proportional to the size of the input, it's usually better to have a few tests with large input rather than many with medium-sized inputs. For example, 100 tests with huge numbers or arrays is usually better than 1000 tests with moderately large arrays/numbers. At the same time, authors should ensure that large inputs are generated and handled in a way that does not have too much negative impact on the test suite.
```